app/layout.tsx

```diff
@@ -2,6 +2,7 @@ import type { Metadata } from "next";
 import { GeistSans } from "geist/font/sans";
 import { Header } from "@/components/header";
 import "@/globals.css";
+import { SocketProvider } from "@/components/socket-provider";
```

Comment:
curious that prettier didn't catch this
```suggestion
import { Header } from "@/components/header";
import { SocketProvider } from "@/components/socket-provider";
import "@/globals.css";
```